spark-cassandra-connector/scala-2.11/src/main/scala/com/datastax/spark/connector/rdd/reader/AnyObjectFactory.scala

```diff
@@ -0,0 +1,194 @@
+package com.datastax.spark.connector.rdd.reader
```

Comment:
Agreed. I brought up the idea of creating the primary trait and branch the changed functions in the 2 branches. I'll push that soon.